### PR TITLE
Adds local OOC

### PR DESCRIPTION
**Pull request by ThatLing**
https://github.com/yogstation13/Yogstation-TG/pull/2636

### CODE REVIEW

**Comment by morrowwolf** [comment]
Anyway to force LOOC to be enabled by default?  Could be that I'm not a new player that does it (like with hotkeys I had to set it to 'L' but I think it's because it was reading hotkeys from my already made hotkey list) but I'd like it enabled by default for new players and want to confirm it is.

File: code/__DEFINES/~yogs_defines/preferences.dm [hunk 1]
```diff
@@ -1 +1,7 @@
-#define DONOR_CHARACTER_SLOTS 6
\ No newline at end of file
+#define DONOR_CHARACTER_SLOTS 6
+
+
+#define CHAT_LOOC	(1<<10)
+
+#undef TOGGLES_DEFAULT_CHAT
+#define TOGGLES_DEFAULT_CHAT (CHAT_OOC|CHAT_DEAD|CHAT_GHOSTEARS|CHAT_GHOSTSIGHT|CHAT_PRAYER|CHAT_RADIO|CHAT_PULLR|CHAT_GHOSTWHISPER|CHAT_GHOSTPDA|CHAT_GHOSTRADIO|CHAT_LOOC)
```
Comment (ThatLing) [inline code/__DEFINES/~yogs_defines/preferences.dm]: This sets LOOC to be enabled by default

Comment (ThatLing) [reply code/__DEFINES/~yogs_defines/preferences.dm]: @morrowwolf

Comment (morrowwolf) [reply code/__DEFINES/~yogs_defines/preferences.dm]: OK I'm happy.

**Pull request merged**